iceoryx_hoofs/test/moduletests/test_container_vector.cpp

```diff
@@ -960,6 +963,84 @@ TEST_F(vector_test, EraseOfMiddleElementCallsDTorAndMove)
     EXPECT_THAT(moveAssignment, Eq(2U));
 }
 
+TEST_F(vector_test, AccessOfNonExistingElementOnEmptyVectorLeadTermination)
```

Comment:
Nitpick: Something for a follow-up PR

```suggestion
TEST_F(vector_test, AccessOfNonExistingElementOnEmptyVectorLeadsToTermination)
```

Same for the other test cases.